project.clj

```diff
@@ -24,7 +24,7 @@
                  [org.cyverse/authy "2.8.0"]
                  [org.cyverse/clojure-commons "2.8.0"]
                  [org.cyverse/kameleon "2.8.0"]
-                 [org.cyverse/mescal "2.8.0"]
+                 [org.cyverse/mescal "2.8.1-SNAPSHOT"]
```

Comment:
I agree with @dennisroberts71 on this one -- obviously we want to avoid snapshot dependencies in releases of services, but I think it's fine to commit them in snapshot versions while testing is ongoing. Since there may easily be more changes to mescal before this work is done, it makes sense to not release it until it's a reasonably complete package of changes (unless a release is needed for some other reason before then, of course).